test/gpu/native/noGpu/SKIPIF

```diff
@@ -0,0 +1 @@
+CHPL_GPU != cpu
```

Comment:
Do we want to do nightly testing for "no gpu"? Probably? Anyway not something for this PR but create a GH issue unless you're convinced that we shouldn't do nightly testing in this mode.

Reply:
https://github.com/Cray/chapel-private/issues/4815